packages/tax/src/services/tax-module-service.ts

```diff
@@ -166,4 +167,145 @@ export default class TaxModuleService<
   ) {
     return await this.taxRateRuleService_.create(data, sharedContext)
   }
+
+  async getTaxLines(
+    items: (TaxTypes.TaxableItemDTO | TaxTypes.TaxableShippingDTO)[],
+    calculationContext: TaxTypes.TaxCalculationContext,
+    @MedusaContext() sharedContext: Context = {}
+  ): Promise<(TaxTypes.ItemTaxLineDTO | TaxTypes.ShippingTaxLineDTO)[]> {
+    const regions = await this.taxRegionService_.list(
+      {
+        $or: [
```

Comment:
curiours to see if this would work instead
```ts
{
  country_code: calculationContext.address.country_code,
  province_code: [null, calculationContext.address.province_code]
}
```

Reply:
yeah I would have expected that to work too, nevermind we can see that later